tests/v2/validation/steve/vai/vai_test.go

```diff
@@ -0,0 +1,545 @@
+//go:build (infra.any || cluster.any || sanity || validation) && !stress && !extended
+
+package vai
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/kubectl"
+	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	coreV1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const scriptURL = "https://raw.githubusercontent.com/brudnak/rancher/brudnak-steve-29/tests/v2/validation/steve/vai/scripts/script.sh"
```

Comment:
Since the script is available locally in the scripts/script.sh file, could we replace the remote URL with : `const scriptPath = "./scripts/script.sh"`

Reply:
I don't think so, because it's being used as a curl command and isn't run locally, the curl command gets executed on the pod

`fmt.Sprintf("curl -k -sSL %s | TABLE_NAME='_v1_Secret_fields' RESOURCE_NAME='%s' sh", scriptURL, secretName)`